dev-libs/udis86/udis86-1.7.2.recipe

```diff
@@ -6,11 +6,12 @@ HOMEPAGE="http://udis86.sourceforge.net/"
 COPYRIGHT="2002-2012, Vivek Thampi"
 LICENSE="BSD (2-clause)"
 REVISION="1"
```

Comment:
```diff
-REVISION="1"
+REVISION="2"
```
(I just realised it would be nice to bump `REVISION` in order to really get rid of the libtool file in the x86_gcc2 package.)